nixos/modules/services/web-apps/vikunja.nix

```diff
@@ -124,10 +123,6 @@ in {
     services.nginx.virtualHosts."${cfg.frontendHostname}" = mkIf cfg.setupNginx {
```

Comment:
Since Vikunja now bundles the frontend with the api, is there any reason to keep the nginx config?

IMHO, people who want to expose Vikunja under a domain or want otherwise customize their hosting experience (which would be a reason to keep nginx) probably have a reverse proxy of sorts set up on their system already and want to integrate with that instead.

Reply:
Probably right, as it's now very easy to setup.